From ab47d5735169c0e9f2dbdb084515b4ecd88b3f54 Mon Sep 17 00:00:00 2001
From: Mooffie <mooffie@gmail.com>
Date: Sun, 5 Jul 2015 17:51:51 +0300
Subject: [PATCH] Ticket #3493: menu selectors: make "User", not "Local", the
 focused button everywhere.

(This completes the previous commit belonging to this ticket.)

Users are more likely to use the "User" button, not the "Local" one.
---
 src/editor/editcmd.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/editor/editcmd.c b/src/editor/editcmd.c
index ea9f49f..a2f2252 100644
--- a/src/editor/editcmd.c
+++ b/src/editor/editcmd.c
@@ -2170,6 +2170,7 @@ edit_load_menu_file (WDialog * h)
     int dir;
     gboolean ret;
 
+    query_set_sel (1);
     dir = query_dialog (_("Menu edit"),
                         _("Which menu file do you want to edit?"), D_NORMAL,
                         geteuid () != 0 ? 2 : 3, _("&Local"), _("&User"), _("&System wide"));
-- 
2.1.4

